railties/lib/rails/test_unit/testing.rake

```diff
@@ -23,13 +23,14 @@ namespace :test do
 
   task run: %w[test]
 
-  desc "Run tests quickly, but also reset db"
+  desc "Runs tests quickly, but also resets the database"
```

Comment:
This runs all tests except `test:system` so a better description might be something like:
```suggestion
  desc "Resets the database, then runs `test`"
```

Reply:
Yes, I like that better.  Though it does sound a bit off when changed to the infinitive form.  But maybe?

```suggestion
  desc "Reset the database and run `test`"
```

Or, additionally expanding \`test\`:

```suggestion
  desc "Reset the database and run `bin/rails test`"
```